modules/org.restlet.test/src/org/restlet/test/ext/apispark/conversion/swagger/v2_0/SwaggerLoader.java

```diff
@@ -27,17 +27,29 @@
 import java.io.File;
 import java.io.IOException;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.wordnik.swagger.models.Swagger;
 import com.wordnik.swagger.util.Json;
 
 public class SwaggerLoader {
 
+    private static ObjectMapper mapper() {
+        ObjectMapper mapper = Json.mapper();
```

Comment:
have you check the source code or the javadoc? I think that mapper() return a static instance